modules/acquia_cms_document/composer.json

```diff
@@ -6,6 +6,9 @@
     "require": {
         "drupal/acquia_cms_common": "^1.9 || ^2.1 || ^3.1"
     },
+    "require-dev": {
+        "drupal/acquia_claro": "^1.3"
+    },
```

Comment:
Why is the Acquia Claro theme required as a development dependency in the Acquia CMS Document module?

Reply:
Failed pipeline: https://git.drupalcode.org/project/acquia_cms_document/-/jobs/1068895#L3315
MR after rebase from 1.x: https://git.drupalcode.org/project/acquia_cms_document/-/merge_requests/2/diffs
Success Pipeline: https://git.drupalcode.org/issue/acquia_cms_document-3397087/-/pipelines/119889

Reply:
got it, thanks.